src/pages/nfts/[address].tsx

```diff
@@ -967,26 +960,27 @@ const NftShow: NextPage<NftPageProps> = ({ marketplace }) => {
                           )}
                           <div className="flex flex-col">
                             <a
-                              href={`https://holaplex.com/profiles/${a.fromWallet}`}
+                              href={`https://holaplex.com/profiles/${a.wallets[0]}`}
                               rel="nofollower"
                               className="text-sm"
                             >
-                              {truncateAddress(a.fromWallet)}
+                              {truncateAddress(a.wallets[0])}
                             </a>
-                            {a.toWallet && (
+                            {a.wallets.length > 1 && (
                               <a
-                                href={`https://holaplex.com/profiles/${a.toWallet}`}
+                                href={`https://holaplex.com/profiles/${a.wallets[1]}`}
                                 rel="nofollower"
                                 className="text-sm"
                               >
-                                {truncateAddress(a.toWallet)}
+                                {truncateAddress(a.wallets[1])}
                               </a>
                             )}
                           </div>
                         </div>
                         <div className="self-center">
                           <span className="sol-amount">
-                            {toSOL(a.price.toNumber())}
+                            {toSOL(parseInt(a.price))}
```

Comment:
This is a good one to note. Since js cant handle the integer size of lamports our scaler::Lamports graphql type returns a string then in the client.ts with apollo map all lamport like fields to Binary with BN js.